docs/updating/migration-to-cdn/angular.md

```diff
@@ -0,0 +1,177 @@
+
+## Migration steps
+
+### Step 1: Remove CKEditor&nbsp;5 imports
+
+If you have any CKEditor&nbsp;5 imports in your Angular components, remove them. For example, remove lines like:
+
+```javascript
+import { ClassicEditor, ... } from 'ckeditor5';
+import { AIAdapter, ... } from 'ckeditor5-premium-features';
+```
+
+### Step 2: Update your Angular components to use CDN
+
+Replace the CKEditor&nbsp;5 NPM package imports with the CDN script imports and use the `loadCKEditorCloud` function to load the CKEditor&nbsp;5 scripts. The `loadCKEditorCloud` function is a part of the `@ckeditor/ckeditor5-angular` package and is used to load CKEditor&nbsp;5 scripts from the CKEditor Cloud service.
+
+**Before:**
+
+```ts
+// app.component.ts
+
+import { Component, ViewEncapsulation } from '@angular/core';
+import { CKEditorModule } from '@ckeditor/ckeditor5-angular';
+import { ClassicEditor, Bold, Essentials, Italic, Mention, Paragraph, Undo } from 'ckeditor5';
+import { SlashCommand } from 'ckeditor5-premium-features';
+
+@Component( {
+	selector: 'app-root',
+	templateUrl: './app.component.html',
+	styleUrls: ['./app.component.css'],
+	encapsulation: ViewEncapsulation.None,
+	imports: [ CKEditorModule ],
+	standalone: true
+} )
+export class AppComponent {
+	title = 'angular';
+
+	public Editor = ClassicEditor;
+	public config = {
+		licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
+		plugins: [ Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo ],
+		toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+		// ... Other configuration ....
+	}
+}
+```
+
+**After:**
+
+```ts
+// app.component.ts
+
+import { Component, ViewEncapsulation } from '@angular/core';
+import { CKEditorModule } from '@ckeditor/ckeditor5-angular';
+
+@Component( {
+	selector: 'app-root',
+	templateUrl: './app.component.html',
+	styleUrls: ['./app.component.css'],
+	encapsulation: ViewEncapsulation.None,
+	imports: [ CKEditorModule ],
+	standalone: true
+} )
+export class AppComponent {
+	title = 'angular';
+
+	public Editor;
+	public config;
+
+	public ngOnInit() {
+		// ADDED
+		loadCKEditorCloud( {
+			version: '43.0.0'
+		} )
+			.then( this._setupEditor.bind( this ) );
+	}
+
+	private _setupEditor( cloud ) {
+		const {
+			ClassicEditor, Bold, Essentials, Italic,
+			Mention, Paragraph, SlashCommand, Undo
+		} = cloud.CKEditor;
+
+		this.Editor = ClassicEditor;
+		this.config = {
+			licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
+			plugins: [ Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo ],
+			toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+			// ... Other configuration ....
+		}
+	}
+}
+```
+
+```html
+// app.component.html
+
+<ckeditor
+	*ngIf="Editor"
+	[editor]="Editor"
+	[config]="config"
+	...
+</ckeditor>
```

Comment:
I would do something like this because the highlight now works incorrectly.
```suggestion
<ckeditor
	*ngIf="Editor"
	[editor]="Editor"
	[config]="config"
	<!-- ... -->
>
</ckeditor>
```